### ISSUE TITLE
Disable / uninstall Flag module

### ISSUE THREAD

**Issue by aaronschachter** [DoSomethingArchive/legacy-website#3677]
~~Blocked by #3652 and #3562~~
- [ ] Disable and uninstall Flag module on all sites.
- [x] Remove Flag module from the profile and all related dependencies
- [ ] Remove Flag module from `drupal-org.make`

**Comment by aaronschachter** [comment]
@sergii-tkachenko Would you be able to take this one when you have a chance? Non urgent, but would be good to get Flag out of the mix since we don't use it anymore.

**Comment by sergiitk** [comment]
Yes, I can do that. Assigning this to myself.

**Comment by aaronschachter** [comment]
@sergii-tkachenko Let's put this one on hold for now... we may end up using Flag down the line.
